static/docs/commands-reference/install.md

```diff
@@ -1,16 +1,43 @@
 # install
 
-Install dvc hooks into the repository
+Install DVC hooks into the Git repository to automate certain common actions
 
 ## Synopsis
 
 ```usage
     usage: dvc install [-h] [-q] [-v]
 ```
 
-## Installed hooks
-- pre-commit : dvc status
-- post-checkout : dvc checkout
+## Description
+
+As designed DVC combines an intelligent data repository with using a regular SCM
+like Git to store code and configuration files. With `dvc install` the two are
+more tightly integrated, causing certain actions to happen automatically. This
+saves the DVC user from having to remember to perform both DVC and SCM actions.
+
+There are two modes of tracking files in a DVC workspace:
```

Comment:
I'm not sure I understand these two points and their connection to everything else to be honest.

Reply:
Reworded